spec/definitions.yaml

```diff
@@ -1137,3 +1141,32 @@ definitions:
           max:
             description: Maximum number of nodes in the pool
             type: integer
+
+  # cluster labels response
+  V5ClusterLabelsResponse:
+    type: object
+    description: Cluster labels
+    additionalProperties: false
+    properties:
+      labels:
+        $ref: "./definitions.yaml#/definitions/V5ClusterLabels"
+
+  # cluster labels
+  V5ClusterLabels:
+    type: object
+    title: Labels object
+    description: Object containing keys with string values representing the labels attached to the cluster
+    additionalProperties:
+      title: Labels
+      description: Key value pairs representing the labels attached to the cluster
```

Comment:
Super nit pick: Would be cool to have some order consistency - does the `description:` field come first / last / second :P

Reply:
Changed it to `type`, `title`, `description`